src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -354,7 +354,7 @@ const NotebookPreviewFrame = ({ notebookName, workspace: { workspace: { googlePr
 
 const JupyterFrameManager = ({ onClose, frameRef, details = {} }) => {
   Utils.useOnMount(() => {
-    Ajax().Metrics.captureEvent(Events.notebookLaunch, { 'Notebook Name': details.notebookName, 'Workspace Name': details.name, 'Workspace Namespace': details.namespace })
+    Ajax().Metrics.captureEvent(Events.notebookLaunch, { 'Notebook Name': details.notebookName, 'Workspace Name': details.name, 'Workspace Namespace': details.googleProject })
```

Comment:
I think the `Metrics` stuff still should capture workspace namespace/name, so this should probably be left as-is.